src/masternode/utils.cpp

```diff
@@ -46,30 +46,33 @@ void CMasternodeUtils::DoMaintenance(CConnman& connman, const CMasternodeSync& m
     }
 
     connman.ForEachNode(CConnman::AllNodes, [&](CNode* pnode) {
-        // we're only disconnecting m_masternode_connection connections
-        if (!pnode->m_masternode_connection) return;
-        if (!pnode->GetVerifiedProRegTxHash().IsNull()) {
-            // keep _verified_ LLMQ connections
-            if (connman.IsMasternodeQuorumNode(pnode)) {
-                return;
-            }
-            // keep _verified_ LLMQ relay connections
-            if (connman.IsMasternodeQuorumRelayMember(pnode->GetVerifiedProRegTxHash())) {
+        if (pnode->m_masternode_probe_connection) {
+            // we're not disconnecting masternode probes for at least a few seconds
+            if (GetSystemTimeInSeconds() - pnode->nTimeConnected < 5) return;
```

Comment:
Is it important that this be not mockable? Maybe we should instead use GetTime<T>?

Reply:
`nTimeConnected` is initialized as not mockable https://github.com/dashpay/dash/blob/master/src/net.cpp#L3770 so I don't think we have any option here

Reply:
So yes it's important it's non-mockable :)